libs/symphonia/src/testing/mod.rs

```diff
@@ -68,9 +91,11 @@ impl Harness {
                 {
                     let pmaker = pmaker.clone();
                     let send_decision = send_decision.clone();
+                    let decision_counter = Arc::clone(&metrics_gatherer);
                     async move {
                         let decision = pmaker.decision().await;
                         send_decision.try_send(decision).unwrap();
+                        // TODO: how do we get the pk of the node who decided?
```

Comment:
@nullchinchilla is this the right place to add the logic to store decision event? I wasn't sure how to get the public key of the node which decided